### PR TITLE
Amp consent int test

### PR DESCRIPTION
**Pull request by paruchurisilpa**
Resolves https://github.com/bbc/simorgh/issues/8810.

**Overall change:**
_A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: cypress/integration/pages/onDemandAudio/tests.js
Comment (andrewscfc) [inline cypress/integration/pages/onDemandAudio/tests.js]: Any idea why this test is on this branch? It should probably be moved to a seperate PR?

File: cypress/integration/specialFeatures/cookieBanner/testsForAMPOnly.js
Comment (andrewscfc) [inline cypress/integration/specialFeatures/cookieBanner/testsForAMPOnly.js:89]: Did you confirm this behaviour is correct/acceptable with @aeroplanejane, I think it differs from canonical where:
- The privacy banner shows and immediately drops a cookie
- Reload the page
- The privacy banner does not show as the cookie has been dropped
?

I think @simonsinclair suggested it may not be possible on amp but just checking it is known to product at the very least.

**Review by andrewscfc — CHANGES_REQUESTED** [review]
Looks good aside from an unrelated test on this PR

File: cypress/integration/pages/onDemandAudio/tests.js
Comment (LilyL0u) [reply cypress/integration/pages/onDemandAudio/tests.js]: As mentioned on Slack, this should be removed from this PR and doesn't need to be put on another one.

**Review by LilyL0u — DISMISSED** [review]

**Comment by paruchurisilpa** [comment]
@LilyL0u removed teh unwanted test now.

**Review by andrewscfc — APPROVED** [review]
Approving assuming the privacy banner behaviour is okay with product as mentioned in the comment 👍

File: cypress/integration/specialFeatures/cookieBanner/testsForAMPOnly.js
Comment (LilyL0u) [inline cypress/integration/specialFeatures/cookieBanner/testsForAMPOnly.js:15]: Is there a reason that the test  it('should have a privacy & cookie banner, which disappears once "accepted" ' now only checks the visibility of the privacy banner? If this is a discussion I missed, and we do want the rest of the steps removed, we should change the name of the test because it no longer accepts the banner.

@paruchurisilpa @andrewscfc

**Review by LilyL0u — COMMENTED** [review]
Is this test supposed to be only checking the visibility of the privacy banner?

File: cypress/integration/specialFeatures/cookieBanner/testsForAMPOnly.js
Comment (andrewscfc) [reply cypress/integration/specialFeatures/cookieBanner/testsForAMPOnly.js:15]: I'm not sure why those steps have been removed, that doesn't seem correct actually good spot

**Review by LilyL0u — APPROVED** [review]

File: src/app/containers/OnDemandHeading/StyledRadioHeadingContainer.jsx
Comment (simonsinclair) [inline src/app/containers/OnDemandHeading/StyledRadioHeadingContainer.jsx]: Please can we remove these unrelated snapshots from this PR before merging into the base branch?

**Pull request merged**
